tests/test_project.py

```diff
@@ -8,3 +10,27 @@ def test_project_folder(cookies):
     assert project.exception is None
     assert project.project.basename == 'my-python-project'
     assert project.project.isdir()
+
+
+def run(command: str, dirpath: os.PathLike) -> subprocess.CompletedProcess:
+    return subprocess.run(shlex.split(command),
```

Comment:
If `command.split(" ")` would also work, i think that's better

Reply:
shlex is a included in Python so does not feel like an extra dep to me and you should keep its docs under your pillow. The shlex is more robust in splitting. However the current commands are simple and str.split will work.

Can you create an issue for this?

Reply:
Made issue #200, marking this as resolved.